### ISSUE TITLE
Kotor 1 Works Only Once with QindieGL Wrapper

### ISSUE THREAD

**Issue by Magmusic** [NVIDIAGameWorks/rtx-remix#289]
Star Wars Knights of the Old Republic

**Describe the bug**
- Remix works one time only with qindieGL 1.0 rev5 wrapper, after game is turned off remix refuses to load with wrapper afterwards. I have tested this by reinstalling game. 
- 
- Remix stops game from loading to start screen.

- When I remove wrapper game starts as normal 

- [Kotor is an Open GL game Remix has no effect without wrapper]

- When I remove Rtx Remix game starts as normal

- Remix version 3.0

- GPU driver -- Device name:     : NVIDIA GeForce RTX 3050 Laptop GPU
info:    Vendor name:     : NVIDIA
info:    Device ID:       : 0x25a2
info:    Vendor ID:       : 0x10de
info:    Driver version   : 546.33.0
- Which game build (and whether it is from Steam/GOG/Other)

**Attach files!**
-
- 
[d3d9.log](https://github.com/NVIDIAGameWorks/rtx-remix/files/13798208/d3d9.log)
[NvRemixBridge.log](https://github.com/NVIDIAGameWorks/rtx-remix/files/13798209/NvRemixBridge.log)
[QindieGL.log](https://github.com/NVIDIAGameWorks/rtx-remix/files/13798210/QindieGL.log)
[swkotor - RTX_d3d9.log](https://github.com/NVIDIAGameWorks/rtx-remix/files/13798211/swkotor.-.RTX_d3d9.log)

  Note, the dumps may contain personally identifiable information.

**To Reproduce**

**Expected behavior**
- Expected game to load remix more than once

**Screenshots**
- If applicable, add screenshots to help explain your problem.
- If it is a visual issue where the raytraced version is wrong/different in comparison to the rasterized version, attach a screenshot of the rasterized version for reference. You can view the rasterized version by disabling raytracing (available in debug and debugOptimized builds).
![Screenshot 2023-12-29 071231](https://github.com/NVIDIAGameWorks/rtx-remix/assets/151014108/acf5276f-ebd1-4c7b-a07c-45ec20998e1a)
![Screenshot 2023-12-29 082136](https://github.com/NVIDIAGameWorks/rtx-remix/assets/151014108/07be4d64-0750-40c7-8eea-4fc3a15a8992)

**Comment by Magmusic** [comment]
ADDED DMP FILE
[NvRemixBridge.exe_20231230_084741.dmp](https://github.com/NVIDIAGameWorks/rtx-remix/files/13798715/NvRemixBridge.exe_20231230_084741.dmp)

**Comment by Magmusic** [comment]
[swkotor ----_d3d9.log](https://github.com/NVIDIAGameWorks/rtx-remix/files/13806028/swkotor.----_d3d9.log)

Compatibility 640x480 enables the game to start but freeze before the title screen

**Comment by NV-DM** [comment]
REMIX-2550 for tracking.

**Comment by NV-DM** [comment]
This should be fixed in the next release. The fix has been merged into Bridge main and can be tested by downloading latest artifacts.
